renovate.json

```
@@ -0,0 +1,53 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":dependencyDashboard",
+    ":semanticCommits",
+    ":semanticCommitScope(deps)",
+    "group:linters",
+    "group:googleapis",
+    "group:goOpenapi",
+    "group:reactMonorepo",
+    "group:reactrouterMonorepo"
+  ],
+  "packageRules": [
+    {
+      "datasources": ["docker"],
+      "updateTypes": ["patch"],
+      "groupName": "docker patch & minor updates"
```

```suggestion
      "groupName": "docker patch updates"
```
 to reflect `updateTypes`